niyodusengaclement/guizaussd
# ft():edit menu functionalities

#### Description of changes made
---

- Add default app menu
- edit menu functionalities


#### How Has This Been Tested?
---

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?
---

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/niyodusengaclement/guizaussd.git), and `cd [the project name]`
- Check out this branch by `git checkout [the branch for this pr]`
- Run `npm i` to install the project dependencies
- Run `npm start` to start the web app in browser.
- .....

#### Checklist:



#### Screenshots
